wcfsetup/install/files/lib/system/box/BoxHandler.class.php

```diff
@@ -185,6 +189,56 @@ public function addBoxToPageAssignments($boxIdentifier, array $pageIdentifiers,
                 $statement->execute([$box->boxID, $page->pageID, $visible ? 1 : 0]);
             }
         }
+
+        $conditionObjectTypeID = ObjectTypeCache::getInstance()->getObjectTypeIDByName(
+            'com.woltlab.wcf.condition.box',
+            'com.woltlab.wcf.page'
+        );
+        $oldCondition = [];
```

Comment:
Why is this an array?

Reply:
The value can be empty, so that is the easiest solution if the default value is an array.